data/map.txt

```diff
@@ -15447,6 +15450,8 @@ system Mesuket
 	link Eneremprukt
 	link Salipastart
 	asteroids "large metal" 1 1.6758
+	minables meretidebris 1 1.98864
+	minables sestordebris 1 2.11643
```

Comment:
Should Mesuket not have the most debris since that's where the majority of the fighting occurs? It would certainly make for a good risk-reward scenario to have lots of debris in Mesuket.

The prevalence of debris so far seems to be tied to the prevalence of other minables in the system, when it should be tied to how much fighting goes on in that system.

Reply:
I increased them to 5 and 4, but i think that making them even higher might be strange since they would be almost the only tipe of asteroids in the system

Reply:
>they would be almost the only tipe of asteroids in the system

That would be the point, would it not?

Reply:
What values do you suggest?

Reply:
If based on conflict, make sure the values are updated via GameEvent when the Sestor vacate their planets. It would be fine to have a gradual phaseout (i.e a few events over several in-game months / weeks).

Reply:
I'm not sure about that, since the lore is based on the fact that the conflict has lasted for hundreds of years; probably even a hundred of years wouldn't be enough to remove all that debris from the space

Reply:
If the payload is balanced to be rewarding while harvested under fire, once the player is not under fire the balance is gone, yes?

Reply:
The asteroids are supposed to be balanced when the player is not under fire, so should I increase the reward to justify the risk pre-sestor shutdown?